developers/weaviate/api/graphql/additional-operators.md

```diff
@@ -306,16 +317,18 @@ Examples:
 
 ### Sorting API
 
-Sorting can be performed by one or more properties. If the values for the first property are identical, Weaviate uses the second property to determine the order, and so on. Thus, the sort operator takes either an object or an array of objects with the two properties described below:
+Sorting can be performed by one or more properties. If the values for the first property are identical, Weaviate uses the second property to determine the order, and so on. 
+
+The sort operator takes either an object, or an array of objects, that describe a property and a sort order.
 
 | Parameter | Required | Type            | Description                                               |
 |-----------|----------|-----------------|-----------------------------------------------------------|
-| `path`    | yes      | `[text]`        | Path to the field to sort by: an array of one element containing the field name. GraphQL supports specifying directly the field name. |
-| `order`   | varies by client       | `asc` or `desc` | Which order to sort by, ascending (default) or descending |
+| `path`    | yes      | `[text]`        | The path to the sort field is an single element array that contains the field name. GraphQL supports specifying the field name directly. |
```

Comment:
I am confused by ``[text]``. Do you mean that it looks something like `["_creationTimeUnix"]` or `["title"]`? If so, I propose `["<property_name>"]`

(Did we call them fields before?) I notice it says "field" on the line here.